extensions/wikia/AdEngine/js/utils/math.js

```diff
@@ -0,0 +1,13 @@
+/*global define*/
+define('ext.wikia.adEngine.utils.math', [
+], function () {
+	'use strict';
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace all tab characters in this file by sequences of white-spaces. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3ATabCharacter)
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Use of JavaScript strict mode may result in unexpected behaviour in some browsers. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AStrictMode)